pv_site_api/main.py

```diff
@@ -355,6 +361,57 @@ def get_pv_estimate_clearsky(site_uuid: str, session: Session = Depends(get_sess
     return res
 
 
+async def get_inverters_helper(session, inverter_ids):
+    if int(os.environ["FAKE"]):
+        return make_fake_inverters()
+
+    client = session.query(ClientSQL).first()
+    assert client is not None
+
+    async with httpx.AsyncClient() as httpxClient:
+        headers = {"Enode-User-Id": client.client_uuid}
+        if not inverter_ids.length:
+            return None
+        inverters_raw = await asyncio.gather(
+            *[
+                httpxClient.get(
+                    f"https://enode-api.production.enode.io/inverters/{id}", headers=headers
+                )
+                for id in inverter_ids
+            ]
+        )
+        inverters = [InverterValues(**inverter_raw.json()) for inverter_raw in inverters_raw]
```

Comment:
This can be moved outside the `with` block since it doesn't use `httpxClient`. All variables in Python variables declared inside the `with` (`inverter_raw`) are still available outside